### PR TITLE
Reduce memory consumption of aiai_euvi_mosaic

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

File: examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py [hunk 1]
```diff
@@ -79,14 +85,16 @@
 ax.set_rlim(0, 1.3)
 
 plt.legend()
+plt.show()
```
Comment (nabobalis) [inline examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py]: I might have removed these from more of the reproduction examples. I thought one show at the end was enough.

File: examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py [hunk 2]
```diff
@@ -48,6 +48,12 @@
 
 maps = sunpy.map.Map(sorted(files))
 
+######################################################################
+# To reduce memory consumption we also downsample these maps before continuing,
+# you can disable this.
+
+maps = [m.resample((1024, 1024)*u.pix) for m in maps]
```
Comment (nabobalis) [inline examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py]: Is it this just this examples we need to do this for?

Comment (Cadair) [reply examples/sunpy_other_packages/reprojection_aia_euvi_mosaic.py]: The main mem issues here seem to be inside reproject's coadd function so I hope it's only this one.

**Comment by nabobalis** [comment]
Did we not backport the reproject stuff?

**Comment by Cadair** [comment]
We did not, they use a few small fixes on master.

**Pull request merged**